tldw_Server_API/tests/AuthNZ_SQLite/test_admin_endpoints_sqlite.py

```
@@ -35,14 +35,42 @@ async def test_admin_endpoints_basic_sqlite(tmp_path):
+    from tldw_Server_API.app.core.AuthNZ.principal_model import AuthPrincipal, AuthContext
 
     async def _pass_admin():
         return {"id": user_id, "role": "admin", "username": "adminuser"}
 
+    async def _principal_override(request=None):  # type: ignore[override]
+        principal = AuthPrincipal(
+            kind="user",
+            user_id=user_id,
+            api_key_id=None,
+            subject="adminuser",
+            token_type="access",
```

Ruff (0.14.6)

Possible hardcoded password assigned to argument: "token_type"

(S106)